src/mongocrypt-kms-ctx.c

```diff
@@ -1314,9 +1483,18 @@ _mongocrypt_kms_ctx_init_gcp_encrypt (
    char *request_string;
    bool ret = false;
 
+   if (!kms) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.